kafka-connect-s3/src/main/java/io/confluent/connect/s3/S3SinkConnectorConfig.java

```diff
@@ -139,6 +142,24 @@
                         ++orderInGroup,
                         Width.LONG,
                         "S3 accelerated endpoint enabled");
+
+      CONFIG_DEF.define(S3_PROXY_HOST,
```

Comment:
The override you're calling here is for a required field. Since not all users will need proxy settings, you should call an override of `define` that includes default values. These can just be placeholders (i.e. `""` and -1).

The current CI test failures are because all the existing tests do not expect to need to set these configs, but the config parsing checks that all required fields are properly filled in.